HISTORY.rst

```diff
@@ -4,7 +4,15 @@ History
 Pending
 -------
 
-* New release notes go here.
+* Conditionally process a response if other middleware is short circuiting. In a
```

Comment:
I rewrote this to start with the explanation that it's a bug fix, and moved some of the more explanatory points into the README so new users put the middleware at the right point too.